src/components/timelapse/TheTimelapse.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="timelapse">
+  <div id="timelapse" v-if="timeLapseMode">
```

Comment:
1. I think the `v-if="timeLapseMode"` check should take place in `src/App.vue` since that is where we control which all subcomponents of our app are displayed or not, so something like:
`<TheTimelapse v-if="timelapseMode" />` at line 9 in `src/App.vue` should work (you can remove the `v-if="mapLoaded"` condition that is present right now, since only when the map loads will the console be visible and the sensors be displayed). You can also then move `timelapseMode` to the state in `src/store/modules/app.js` since it makes sense for our "app" to be in time-lapse mode or not.
2. For the sake of being consistent, let's name it `timelapseMode` and not `timeLapseMode`.